seqr/views/apis/family_api_tests.py

```diff
@@ -5,31 +5,69 @@
 from django.test import TestCase
 from django.urls.base import reverse
 
-from seqr.views.apis.family_api import update_family_pedigree_image, update_family_assigned_analyst, update_family_fields_handler, update_family_analysed_by, edit_families_handler
+from seqr.views.apis.family_api import update_family_pedigree_image, update_family_assigned_analyst, \
+    update_family_fields_handler, update_family_analysed_by, edit_families_handler, delete_families_handler, receive_families_table_handler
 from seqr.views.utils.test_utils import _check_login
 
 FAMILY_GUID = 'F000001_1'
+FAMILY_GUID2 = 'F000002_2'
 
 PROJECT_GUID = 'R0001_1kg'
 EMPTY_PROJECT_GUID = 'R0002_empty'
 
+FAMILY_ID_FIELD = 'familyId'
+PREVIOUS_FAMILY_ID_FIELD = 'previousFamilyId'
+
+
 class ProjectAPITest(TestCase):
-    fixtures = ['users', '1kg_project', 'reference_data']
-    multi_db = True
+    fixtures = ['users', '1kg_project']
+
     #  TODO test other family api methods
 
     def test_edit_families_handler(self):
         url = reverse(edit_families_handler, args=[PROJECT_GUID])
         _check_login(self, url)
 
-        # send request
-        req_values = {'families': [{'familyGuid': FAMILY_GUID, 'description': 'Test'}]}
-        response = self.client.post(url, content_type='application/json', data=json.dumps(req_values))
+        # send request with a "families" attribute
+        req_values = {
+            'families': [
+                {'familyGuid': FAMILY_GUID, 'description': 'Test description 1'},
+                {PREVIOUS_FAMILY_ID_FIELD: '2', 'description': 'Test description 2'},
```

Comment:
So the previous ID field is designed to be used to change the ID, so the usage here is not actually testing a realistic use-case. You should have the previous ID as well as a new ID, and then you should check in the response that the ID for this individual has been successfully updated

Reply:
Okay. Updated.